packages/apps/job-launcher/server/src/common/utils/storage.ts

```diff
@@ -59,15 +63,21 @@ export async function listObjectsInBucket(url: URL): Promise<string[]> {
       let nextContinuationToken: string | undefined;
       const baseUrl = `${url.protocol}//${url.host}`;
       do {
-        const response = await axios.get(
-          `${baseUrl}?list-type=2${
+        let requestOptions = `${baseUrl}`;
+
+        if (url.hostname !== 'localhost') {
+          requestOptions += `?list-type=2${
             nextContinuationToken
               ? `&continuation-token=${encodeURIComponent(
                   nextContinuationToken,
                 )}`
               : ''
-          }${url.pathname ? `&prefix=${url.pathname}` : ''}`,
-        );
+          }${url.pathname ? `&prefix=${url.pathname}` : ''}`;
+        } else {
+          requestOptions += url.pathname ? url.pathname : '';
```

Comment:
In #1861 you have this as: 

> requestOptions += '${url.pathname ? '${url.pathname}' : ''}?list-type=2${
>             nextContinuationToken
>               ? '&continuation-token=${encodeURIComponent(
>                   nextContinuationToken,
>                 )}'
>               : ''
>           }';

Which is the right one?

Reply:
They are right both, `requestOptions += url.pathname ? url.pathname : '';` for minio client on localhost

Reply:
As I can see by date, this should be correct

Reply:
```
requestOptions += `${url.pathname ? `${url.pathname}` : ''}?list-type=2${
            nextContinuationToken
              ? `&continuation-token=${encodeURIComponent(
                  nextContinuationToken,
                )}`
              : ''
          }`;
```

Reply:
I will update